src/Command/Generate/PluginConditionCommand.php

```diff
@@ -97,9 +97,9 @@ protected function interact(InputInterface $input, OutputInterface $output)
     {
         $io = new DrupalStyle($input, $output);
 
-        $entity_manager = $this->getEntityManager();
+        $entityTypeRepository = $this->getService('entity_type.repository');
```

Comment:
@jmolivas can we add a new method entityTypeRepository() to ContainerAwareCommand class ? are we maintaining Backward compatibility ?